crew

```diff
@@ -843,8 +846,8 @@ def resolve_dependencies_and_build
   ensure
     #cleanup
     unless @opt_keep
-      FileUtils.rm_rf Dir.glob("#{CREW_BREW_DIR}/*")
-      FileUtils.mkdir_p "#{CREW_BREW_DIR}/dest" #this is a little ugly, feel free to find a better way
+      FileUtils.rm_rf "#{Dir.glob('#{CREW_BREW_DIR}/*')}#{@fileutils_verbose}"
```

Comment:
This syntax does not look correct.  The `#{...}` wrapper is for variables, not functions.